src/components/layout/OrgHeader.tsx

```diff
@@ -36,12 +36,12 @@ const OrgHeader = ({ org } : OrgHeaderProps) : JSX.Element => {
                     <View marginX="size-200">
                         <Link>
                             <NextLink href="/">
-                                <a><Msg id="components.layout.orgHeader.edit_link"/></a>
+                                <a><Msg id="layout.orgHeader.editLink"/></a>
```

Comment:
This file illustrates an interesting challenge when naming messages. The message on this line is called `editLink`, while the one on line 44 is called `unfollowButton`, i.e. *link* vs *button*.

The difference between the two is mostly visual, but propagates to the naming of messages. Maybe a better naming scheme should just be `edit` and `unfollow`. To indicate that they are both actions (and not labels), maybe add an `actions` part to the ID? E.g.:

* `layout.orgHeader.actions.edit`
* `layout.orgHeader.actions.unfollow`